packages/xod-client/src/debugger/actionTypes.js

```diff
@@ -7,3 +7,5 @@ export const DEBUGGER_LOG_CLEAR = 'DEBUGGER_LOG_CLEAR';
 
 export const DEBUG_SESSION_STARTED = 'DEBUG_SESSION_STARTED';
 export const DEBUG_SESSION_STOPPED = 'DEBUG_SESSION_STOPPED';
+
+export const SAVE_DEBUGGER_NODE_IDS_MAP = 'SAVE_DEBUGGER_NODE_IDS_MAP';
```

Comment:
Actions are facts. Looks like a Redux misuse. The effect should be applied as a reaction to something like `DEBUG_SESSION_STARTED`.